cltk/corpus/hebrew/alphabet.py

```diff
@@ -0,0 +1,13 @@
+"""The Hebrew alphabet.
+
+References:
+* https://en.wikipedia.org/wiki/Hebrew_numerals
+* https://www.omniglot.com/writing/hebrew.htm
+* https://www.duolingo.com/comment/7253301/Hebrew-Time-8-The-Hebrew-Vowels
+"""
+
+DIGITS = ['א','ב','ג','ד','ה','ו','ז','ח','ט','י','יא‬','יב','יג','יד‬','ט״ו‬','ט״ז‬','יז‬','יח‬','יט‬','כ','ל','מ','נ','ס','ע','פ','צ','ק','ר', 'ש', 'ת','ך‬','ם‬','ן‬','ף‬','ץ‬','א׳‬','ב׳‬','ה׳‬','א׳י‬	','א׳ק‬','א׳א׳‬	']
+
+ALPHABET = ['ז', 'ו', 'ה', 'ד', 'ג', 'בּ/ב', 'א', 'מ', 'ל', 'ך', 'כּ/כ', 'י', 'ט', 'ח', 'ף', 'פּ/פ', 'ע', 'ס', 'ן', 'נ', 'ם', 'ת', 'שׁ/שׂ', 'ר', 'ק', 'ץ', 'צ', "'ת", "'צ", "'ע", "'ח", "'ז", "'ד", "'ג"] 
```

Comment:
Alphabetical order is not respected. You used this web site <https://www.omniglot.com/writing/hebrew.htm>, but you read characters from left to right, whereas it is from right to left. As the order matters in a list, a change is requested for that. Moreover, you should use Biblical Hebrew, and not the Modern Hebrew which is a reconstructed language.